ecosystem/sep-0024.md

```diff
@@ -935,5 +937,6 @@ There is a small set of changes when upgrading from SEP-6 to SEP-24.
 * Solar wallet: https://solarwallet.io
 
 ## Changelog
+* `v2.3.0`: Add `lang` field to GET `/transactions` and `/transaction`. Make `lang` field definition consistent. ([#1191](https://github.com/stellar/stellar-protocol/pull/1191))
```

Comment:
Suggest change to lead with the larger change, and be explicit about what is changing:
```suggestion
* `v2.3.0`: Change `lang` format from [ISO639-1] to [RFC4646] which is a superset of [ISO639-1]. Add `lang` field to GET `/transactions` and `/transaction`. ([#1191](https://github.com/stellar/stellar-protocol/pull/1191))
```

Reply:
Changed.